### PR TITLE
leds!

### PR DESCRIPTION
**Pull request by jasonballinger**

### CODE REVIEW

**Review by PSlape2 — APPROVED** [review]

**Comment by PSlape2** [comment]
Looks good, can I get a review on my additons?

File: src/main/java/frc/robot/subsystems/LEDSubsystem.java [hunk 1]
```diff
@@ -34,25 +53,32 @@ public LEDSubsystem() {
         m_led.start();
     }
 
-    public void setOrange() {
-        setRGB(255, 40, 0); // Orange
+    public void setColor(Color color) {
```
Comment (jasonballinger) [inline src/main/java/frc/robot/subsystems/LEDSubsystem.java:56]: !

**Review by jasonballinger — COMMENTED** [review]
rumble command looks good tho

File: src/main/java/frc/robot/subsystems/LEDSubsystem.java (hunk 1 above)
Comment (ArshansGithub) [reply src/main/java/frc/robot/subsystems/LEDSubsystem.java:56]: Sir yes Sir

File: src/main/java/frc/robot/commands/LEDCommand.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        this.intakeDeploySupplier = intakeDeploySupplier;
+        this.shooterSupplier = shooterSupplier;
+        this.timeLastBlink = System.currentTimeMillis();
+
+        addRequirements(ls);
+    }
+
+    @Override
+    public void execute() {
+        if(DriverStation.isDisabled() || DriverStation.getAlliance() == null) {
```
Comment (ArshansGithub) [inline src/main/java/frc/robot/commands/LEDCommand.java:41]: DriverStation.getAlliance() == DriverStation.Alliance.Invalid

add this aswell

**Review by ArshansGithub — APPROVED** [review]
everything is good just needs to be tested

File: src/main/java/frc/robot/commands/LEDCommand.java (hunk 2 above)
Comment (jasonballinger) [reply src/main/java/frc/robot/commands/LEDCommand.java]: does not exist

**Pull request merged**